proposals/init.md

```diff
@@ -71,8 +71,8 @@ class Student
 }
 ```
 
-An instance property containing an `init` accessor is considered settable in
-the following circumstances:
+An instance property containing an `init` accessor is considered settable outside
+a local function or lambda in the following circumstances:
```

Comment:
The wording here is somewhat ambiguous: it doesn't explicitly forbid local functions or lambdas. I would instead phrase it as:

```suggestion
An instance property containing an `init` accessor is considered settable in
the following circumstances, except when in a local function or lambda:
```

As a side note, this might be a good argument for `init` methods, and allowing applying that to local functions :).

Reply:
I like it. Thanks